packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -4494,7 +4489,7 @@ typedef WidgetBuilder = Widget Function(BuildContext context);
 ///
 ///  * [WidgetBuilder], which is similar but only takes a [BuildContext].
 ///  * [TransitionBuilder], which is similar but also takes a child.
-typedef IndexedWidgetBuilder = Widget Function(BuildContext context, int index);
+typedef IndexedWidgetBuilder = Widget? Function(BuildContext? context, int index);
```

Comment:
all builders should return non-nullable Widgets

Reply:
also context on builders should always be non-nullable

Reply:
Done